packages/frontend/src/app/filters/filters.component.ts

```diff
@@ -97,11 +101,27 @@ export class FiltersComponent implements OnInit {
     return options;
   }
 
+  private tryParseOption(option: string, options: Option[]): void {
+    const optionObj = JSON.parse(option);
+    const optionKeys = Object.keys(optionObj);
```

Comment:
the edge case of `optionKeys.length === 0` would be that we have an empty object `{}` as the filter?

might be worth adding a comment.

Reply:
Yep, `Object.keys({}).length  === 0`.